packages/manager/src/views/WebPropertyEnvPage/components/EditAccess/EditMemberAccess.tsx

```diff
@@ -0,0 +1,206 @@
+import { useAddPermission, useDeleteMember } from '@app/services/rbac';
+import { toPascalCase } from '@app/utils/toPascalConvert';
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionToggle,
+  Button,
+  Checkbox,
+  Split,
+  SplitItem
+} from '@patternfly/react-core';
+import { TableComposable, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+import React, { useState } from 'react';
+import toast from 'react-hot-toast';
+interface Props {
+  onClose: () => void;
+  editMemberName: string;
+  propertyIdentifier: string;
+  memberList: any;
+}
+type ColumnNames = {
+  NAME: string;
+  APIKEY_CREATION: string;
+  APIKEY_DELETION: string;
+  PERMISSION_CREATION: string;
+  PERMISSION_DELETION: string;
+  ENV_CREATION: string;
+  ENV_SYNC: string;
+  [key: string]: string;
+};
+type GroupItem1 = {
+  email: string;
+  name: string;
+  role: string;
+  PERMISSION_CREATION: boolean;
+  PERMISSION_DELETION: boolean;
+  ENV_SYNC: boolean;
+  ENV_CREATION: boolean;
+  APIKEY_DELETION: boolean;
+  APIKEY_CREATION: boolean;
+};
+export const EditMemberAccess = ({
```

Comment:
Can you please add a todo comment here. 

Something like this.

```js
// TODO(akhilmhdh): Migrate to react-hook-form later. Removes a lot of boilerplate in this page
```

Reply:
done